cli/src/main/resources/help/alias.adoc

```diff
@@ -0,0 +1,27 @@
+ALIAS(1)
+=======
+:doctype: manpage
+
+
+NAME
+----
+alias - creates aliases for command shortcuts.
+
+
+SYNOPSIS
+--------
+*alias* ['ALIAS']=['COMMAND']
+
+
+EXAMPLES
+--------
+`alias q=quit` +
+Creates `q` as an alias for the `quit` command.
+
+`unalias q` +
```

Comment:
not sure how useful is to have an "unalias" example here.
You can display the aliases defined if you don't use any argument:
```
[disconnected]> alias q=quit
[disconnected]> alias co=connect
[disconnected]> alias
alias co='connect'
alias q='quit'
```

So, it would be more useful a listing example
```
`alias` + 
List all defined alias
```

Reply:
perfect. thanks @pruivo